### DISCUSSION TITLE
Use case: grid size and position solver

### DISCUSSION THREAD

**Discussion by rpdelaney** [redruin1/factorio-draftsman#44] [Ideas]
There may be a way to use draftsman or something like it to take a blueprint and solve for grid size and position settings to tile it. (I am awful at this, it's like kryptonite to me)

**Comment by redruin1** [comment]
I too have always found this annoying/unintuitive. I imagine you're proposing a method that would automatically calculate the dimensions of the blueprint and then set the grid according to it?

**Reply by rpdelaney** [reply]
Yeah, although I'm not yet sure how. I imagine draftsman _might_ be able to do it for me, once the validation rules are relaxed. But I'm not sure how complex it would have to be, to be better than my current workflow for figuring this out in an editor extensions sandbox.
